push-image

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -eu
+
+# Push the 'cli:5' image to Dockerhub when on the 'master' branch
+
+cd "$(git rev-parse --show-toplevel)"
+
+IMAGE='cyberark/conjur-cli'
+
+function tag_and_push() {
+    local image="$1"
+    local tag="$2"
+    local description="$3"
+
+    echo "TAG = $tag, $description"
+
+    docker tag "$image" "$image:$tag"
+    docker push "$image:$tag"
+}
+
+if [[ "$BRANCH_NAME" == 'master' ]]; then
+    bare_tag='5'
+    latest_tag='5-latest'
+    stable_tag="5-$(cat VERSION)-stable"
```

Comment:
I'm okay with making `5-latest` and `latest` be the same thing. I don't really want to push `$(cat version)` because then you'd have `5.5.6` and `5` be two totally different things. We don't necessarily need `stable` at the end.

VERSION is added to this branch now.